spec/models/person_spec.rb

```diff
@@ -58,11 +58,18 @@
     end
     context "existing record" do
       it "updates record with appropriate values" do
-        FactoryGirl.create(:person, email: 'user@example.com')
+        person = FactoryGirl.create(:person, email: 'user@example.com')
         hash = { email: 'user@example.com',
                  phone: '555-555-1111' }
-        person = Person.create_or_update(hash)
-        expect(person.slice(*hash.keys).values).to eq hash.values
+        Person.create_or_update(hash)
+        expect(person.reload.slice(*hash.keys).values).to eq hash.values
+      end
+      it "finds user based on uuid, if present" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.